applications/asset_tracker_v2/tests/nrf_cloud_codec_mocked_cjson/src/nrf_cloud_codec_mocked_test.c

```diff
@@ -0,0 +1,576 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <unity.h>
+#include <stdbool.h>
+
+#include "cloud_codec.h"
+#include <cJSON_os.h>
+#include <errno.h>
+
+#include "mock_json_helpers.h"
+#include "mock_date_time.h"
+#include "mock_cJSON_to_mock.h"
+#include "float.h"
+
+
+struct cloud_codec_data codec;
+int ret;
+
+void setUp(void)
+{
+	__wrap_cJSON_Init_Ignore();
+	ret = cloud_codec_init(NULL, NULL);
+	TEST_ASSERT_EQUAL(0, ret);
+
```

Comment:
Missing `mock_x_Init();` functions for mocked APIs

Reply:
Good catch!